aequilibrae/project/network/node.py

```diff
@@ -76,6 +79,7 @@ def renumber(self, new_id: int):
             *new_id* (:obj:`int`): New node_id
         """
 
+        new_id = int(new_id)
         if new_id == self.node_id:
             raise ValueError('This is already the node number')
```

Comment:
Should this maybe be a warning and a do-nothing return?

Reply:
Better indeed.